setup.py

```diff
@@ -1,12 +1,5 @@
 import sys
-from distutils.core import setup
-
-if sys.version_info[:2] < (2, 5) or sys.version_info[0] > 2:
```

Comment:
Version check is added back. The minimum version allowed is changed to 2.6. 
Classifiers is also updated. 

Also added "install_requires" when calling setup to ensure past package is installed.